src/app/shared/mangoQueries.ts

```diff
@@ -21,3 +21,14 @@ export function findAllDocuments(selector, query) {
     }
   `);
 }
+
+export function findMultiDocuments(selectors, fields, limit, skip) {
+  return JSON.parse(`
+    {
+      "selector": ${JSON.stringify(selectors[0])},
```

Comment:
Why are we using the first element in the selectors array?  In general, what is the format we are expecting for the arguments for `findMultiDocuments`?

Reply:
it should be json object